src/metabase/upload.clj

```diff
@@ -73,6 +73,11 @@
 
 (def ^:private currency-regex "Supported currency signs" #"[$€£¥₹₪₩₿¢\s]")
 
+(defn- with-parens
+  "Returns a regex that matches the argument, with or without surrounding parentheses."
+  [number-regex]
+  (re-pattern (str "(" number-regex ")|(\\(" number-regex "\\))")))
```

Comment:
why do we need a version with escaped parens?

Reply:
Ah I see. This is regex so unescaped parens denote capturing groups, and the escaped parens match actual parens in the string. 

This should still work if we remove the capturing groups, and it's easier to read:
```suggestion
  (re-pattern (str number-regex "|\\(" number-regex "\\)")))
```